index.js

```diff
@@ -67,6 +67,45 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener('click', function(event) {
+    toggleColor(event.target)
+})
+
 /***** Deliverable 2 *****/
 
+const button = document.querySelector('button.like-button')
+
+button.addEventListener('click', function(){
+
+    likes.textContent = `${traveler.likes++} likes`
+   // const currLikes = parseInt(likes.textContent)
+   // likes.textContent = `${currLikes + 1} likes`
+})
```

Comment:
First, you find the like button element on the page. Then, you add an event listener to the like button passing in the event type and callback function as arguments. Great work!